Parsers/ASimWebSession/Parsers/ASimWebSessionCiscoMeraki.yaml

```diff
@@ -1,7 +1,7 @@
 Parser:
   Title: Web Session ASIM filtering parser for Cisco Meraki
-  Version: '0.1'
-  LastUpdated: Jun 20 2023
+  Version: '0.2.1'
+  LastUpdated: May 31 2024
```

Comment:
"LastUpdated" property exists but is not correct format. The expected format is, for example, "Jun 29, 2024"

Reply:
done